Doc/library/unittest.mock-examples.rst

```diff
@@ -300,28 +298,29 @@ Mocking asynchronous context manager
 Since Python 3.8, ``AsyncMock`` and ``MagicMock`` have support to mock
 :ref:`async-context-managers` through ``__aenter__`` and ``__aexit__``.
 By default, ``__aenter__`` and ``__aexit__`` are ``AsyncMock`` instances that
-return an async function.
+return an async function. So you can do:
 
-    >>> class AsyncContextManager:
-    ...     async def __aenter__(self):
-    ...         return self
-    ...     async def __aexit__(self, exc_type, exc, tb):
-    ...         pass
-    ...
-    >>> mock_instance = MagicMock(AsyncContextManager())  # AsyncMock also works here
     >>> async def main():
-    ...     async with mock_instance as result:
-    ...         pass
-    ...
+    ...     async with MagicMock() as async_context_manager: # Replacing MagicMock with AsyncMock would also work
+    ...         print('Inside async context manager')
+    >>> asyncio.run(main())
+    Inside async context manager
+
+This would not work with Mock class, though, because it doesn't have the pre-assigned ``__aexit__`` and ``__aenter__`` methods.
```

Comment:
I'm not sure this needs an example, the sentence is enough.
(Also the line is too long.)